content/applications/sales/sales/invoicing/time_materials.rst

```diff
@@ -1,136 +1,370 @@
+added by the employees or by the person who created the sales order.
+
+To add an employee, and the time spent working on the task, click :guilabel:`Add a line` in the
+:guilabel:`Timesheets` tab. Then, select the appropriate :guilabel:`Date` and :guilabel:`Employee`.
+There is also the option to add a brief description of the work done during this time in the
+:guilabel:`Description` column, but it's not required.
+
+Lastly, enter the amount of time worked on the task in the :guilabel:`Hours Spent` column, and click
+away to complete that line in the :guilabel:`Timesheets` tab.
+
+.. note::
+   The time entered in the :guilabel:`Hours Spent` column is immediately reflected in the
+   :guilabel:`Allocated Time` field (located near the top of the task form) as a percentage, which
+   reflects how much of the total allocated work hours have been done so far.
+
+   That same information is found as numerical hours in the :guilabel:`Hours Spent` and
+   :guilabel:`Remaining Hours` fields, located at the bottom of the :guilabel:`Timesheets` tab.
+
+   .. image:: time_materials/timesheets-tab-on-task.png
+      :align: center
+      :alt: How the Timesheets tab appears on a task form in Odoo Sales and Odoo Project.
+
+Repeat this process for however many employees and hours have been worked on the project.
+
+Invoice time spent
+==================
+
+Once all the necessary employees and time spent have been added to the project task, return to the
+sales order to invoice the customer for those hours. To do that, either click the :guilabel:`Sales
+Order` smart button at the top of the task form, or return to the sales order via the breadcrumb
+links, located in the upper-left of the screen.
+
+Back on the sales order form, the time that was added to the task is reflected in the
+:guilabel:`Order Lines` tab (in the :guilabel:`Delivered` column) and in the new :guilabel:`Recorded
+Hours` smart button at the top of the sales order.
+
+To invoice the customer for time spent on the project, click :guilabel:`Create Invoice`, and select
+:guilabel:`Regular invoice` from the :guilabel:`Create invoices` pop-up window. Then, click
+:guilabel:`Create Draft Invoice`.
+
+Doing so reveals a :guilabel:`Customer Invoice Draft`, clearly showing all the work that's been done
+in the :guilabel:`Invoice Lines` tab.
+
+.. tip::
+   Pay attention to the :guilabel:`Analytic Distribution` column in the :guilabel:`Customer
+   Invoice`, as that information is necessary to ensure other time/material invoicing tasks are
+   completed properly and accurately.
+
+   .. image:: time_materials/invoice-lines-time.png
+      :align: center
+      :alt: Invoice draft showing time spent on sales order in Odoo Sales.
+
+Click :guilabel:`Confirm` to confirm the invoice and continue with the invoicing process.
+
+.. seealso::
+   - :doc:`/applications/sales/sales/invoicing/invoicing_policy`
```

Comment:
No need for bullet on a single item in seealso, and update doc path to be relative please
```suggestion
   :doc:`invoicing_policy`
```